WBooks.xcodeproj/xcuserdata/nmanograsso.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -19,21 +19,5 @@
             landmarkType = "7">
```

Comment:
so just you know, everything under `/xcuserdata` can (and should) be ignored in our repositories (it doesn't matter right now but when you are in a project remember this)